tests/foreman/cli/test_contentviews.py

```diff
@@ -250,6 +250,44 @@ def test_cv_delete(self):
         with self.assertRaises(CLIReturnCodeError):
             ContentView.info({'id': con_view['id']})
 
+    def test_delete_cv_files(self):
+        """@Test: Delete content view and verify it was actually deleted from
+        hard drive.
+
+        @Feature: Content View
+
+        @Assert: Content view was deleted and pulp folder doesn't contain
+        content view files anymore
+
+        """
+        # Create a content view and publish it
+        content_view = make_content_view({'organization-id': self.org['id']})
```

Comment:
Will this not create an empty content view? It will be great to add a real repo for the test to make more sense?